subprojects/packagefiles/cwalk/meson.build

```diff
@@ -0,0 +1,15 @@
+project('cwalk', 'c',
+  license: 'MIT',
+  meson_version: '>= 0.57.0'
```

Comment:
pretty sure you can reduce to 0.49.0

Reply:
..\subprojects\cwalk-1.2.7\meson.build:15: WARNING: Project targets '>= 0.49.0' but uses feature introduced in '0.50.0': include_directories kwarg of type string. Use include_directories('include') instead